test-apps/ui-test-app/src/backend/main.ts

```diff
@@ -31,6 +34,9 @@ import { initializeWeb } from "./web/BackendServer";
     const iModelClient = new  IModelsClient({ api: { baseUrl: `https://${process.env.IMJS_URL_PREFIX ?? ""}api.bentley.com/imodels`}});
```

Comment:
Looks like some merge was messed up here as we lost the IModelsClient configured.

Reply:
Not sure I'm following.  Looking at master everything seems the same except for the registerImpl call I added.

Reply:
Hm looks like `master` lost this change as well... Not sure what happened but I'll take a look.

Reply:
ignore that last message...